### PR TITLE
Blob Fake Announcements are now followed by a clarification 60 to 180 seconds later.

### PR DESCRIPTION
**Pull request by BurgerLUA**
Draft because untested.

## About The Pull Request

Blob Fake Announcements are now followed by a clarification 20 to 40 seconds later.

## Why It's Good For The Game

Prevents issues of Dynamic fucking up and putting everyone on red alert status for no real reason (fucking over antags).

## Changelog

:cl: BurgerBB
balance: Blob Fake Announcements are now followed by a clarification 60 to 180 seconds later.
/:cl:

### CODE REVIEW

**Comment by BurgerLUA** [comment]
Might increase the time. 20 to 40 seconds seems kind of short. Maybe 2-3 minutes?

**Comment by StrangeWeirdKitten** [comment]
> Might increase the time. 20 to 40 seconds seems kind of short. Maybe 2-3 minutes?

I'm just going to add that the blob announcements are not ICES, or Dynamic, or a random event. They're litterally traitors manually inputting the announcements into the field.

**Comment by Majkl-J** [comment]
Admin pressed the funny button on a calm round and suddenly we have a draft change

**Comment by BurgerLUA** [comment]
> > Might increase the time. 20 to 40 seconds seems kind of short. Maybe 2-3 minutes?
> 
> I'm just going to add that the blob announcements are not ICES, or Dynamic, or a random event. They're litterally traitors manually inputting the announcements into the field for 3 TC per announcement. You can make it five TC and that may help a little or a better solution is to have the traitor announcement faker not have certain announcement sounds.

Out of the scope of this PR.

**Comment by aKromatopzia** [comment]
> Might increase the time. 20 to 40 seconds seems kind of short. Maybe 2-3 minutes?

honestly 5 minutes is a good while, long enough for the station to have fanned out to inconclusively search

File: modular_zubbers/code/modules/events/ghost_role/blob.dm [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+				unlucky_victim_fatty_score = possible_fatty_score
+
+		addtimer(CALLBACK(src,PROC_REF(lose_some_weight_fatty),unlucky_victim),rand(20 SECONDS,40 SECONDS))
+
+/datum/round_event/ghost_role/blob/proc/lose_some_weight_fatty(victim)
+	priority_announce(
+		"It appears that the level 5 biohazard aboard [station_name()] was a false alarm, as our sensors appeared to have mistakenly labeled [victim] as a level 5 biohazard. All crewmembers are reminded to practice healthy eating habbits as part of Nanotrasen's Safe and Healthy Eating program.",
+		"Biohazard Alert",
+		ANNOUNCER_OUTBREAK5
+	)
```
Comment (StrangeWeirdKitten) [inline modular_zubbers/code/modules/events/ghost_role/blob.dm:39]: Ok this is pretty funny

File: modular_zubbers/code/modules/events/ghost_role/blob.dm [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+			var/possible_fatty_score = possible_fatty.nutrition
+			if(possible_fatty.has_quirk(/datum/quirk/overweight))
+				possible_fatty_score = possible_fatty_score * 1.5 + 1000
+
+			if(possible_fatty_score > unlucky_victim_fatty_score)
+				unlucky_victim = "[possible_fatty]" //We don't use .name here because we want "The" to be affixed to it if needed.
+				unlucky_victim_fatty_score = possible_fatty_score
+
+		addtimer(CALLBACK(src,PROC_REF(lose_some_weight_fatty),unlucky_victim),rand(20 SECONDS,40 SECONDS))
```
Comment (StrangeWeirdKitten) [inline modular_zubbers/code/modules/events/ghost_role/blob.dm:32]: Yeah make this a few minutes

**Comment by Majkl-J** [comment]
> > Prevents issues of Dynamic fucking up and putting everyone on red alert status for no real reason
> 
> Its a paranoia simulator

The paranoia is fun until you get sec using their newly authorized weapons to burst down tots

**Comment by StrangeWeirdKitten** [comment]
This is fine. Bug me if I forget to merge this.

**Pull request merged**